k8s/go/pkg/resolver/resolver.go

```diff
@@ -12,7 +12,6 @@ import (
 	"strings"
 
 	"github.com/bazelbuild/rules_docker/container/go/pkg/compat"
```

Comment:
what about this one, should it be inlined too? I guess you'll have a later PR to drop the go_repository that fetches rules_docker code. I don't know what is the maintenance story for rules_k8s so maybe we should wait-and-see if we get any responses.

Reply:
Yeah, maybe that one should also be inlined as part of this change. I had held off since that particular import needs to be translated to understand the OCI image format rather than the format rules_docker is outputting today. But that translation could easily be done in a followup PR after inlining in this one